### ISSUE TITLE
Should it be 'initialise' or 'initialize'

### ISSUE THREAD

**Issue by itskingori** [Intellection/docker-vault-init#5]
Nit-pick alert! ⚠️ 

I noticed this in the beginning but there were so many other more important stuff we needed to figure out. I'm leaning on the later and I don't have a very good argument for it other than:

1. Vault is an American company 😅 
2. The `z` variant is more popular in the programming world e.g. Ruby with `def initialize; end`.

**Comment by hadrianvalentine** [comment]
I agree that the American spelling is more common. There are likely many reasons for this (like how many who are non native speakers often default to American). That said, there are still occurrences of the [British variant on GitHub](https://github.com/search?q=initialise). Also, while Vault is an American company, Intellection/Zappi is a British company. It seemed appropriate given that that those of use who have both written code and offered reviews on this project happen to speak British English.

Of course, these are just the thoughts swirling around in my own head. Interested to hear more opinions on this. As this is now public we may even get some outside commentary.
